test/chplcheck/ConsecutiveDecls.chpl

```diff
@@ -0,0 +1,22 @@
+module ConsDecls {
+  var real1: real;
+  var real2: real;
+
+  const constReal1: real;
+  const constReal2: real;
+
+  const constReal3:  real;
+  const constIntBetween:  int;
+  const constReal4:  real;
+
+  const cr1: real;
+  var vr1: real;
+  const cr2: real;
+
+  const ref crr1: real = cr1;
+  const between: real;
+  const ref crr2: real = cr1;
+  const ref ref1: real = cr1;
+  const ref ref2: real = cr1;
+  const ref ref3: real = cr1;
```

Comment:
More things to test: variables that have `@annotations` or `pragma "whatver"`. In this case, we shouldn't warn:

```
@chpldoc.nodoc
var x, y = 42;
```

Is not the same as

```
@chpldoc.nodoc
var x = 42;
var y = x;
```

Reply:
Ditto `extern` and other modifiers on variables.

Reply:
I would expect this case to warn, same with pragmas